### ISSUE TITLE
New Update???

### ISSUE THREAD

**Issue by Kitty035** [ZaneYork/SMAPI-Android-Installer#381]
I understand that you do this when you can, but it has been a good month since the last update and it appears that no more work is being done.  There are some major issues with the alpha version now.  Just wondering if you are planning to fix or just wait for the next game update.  Thank you for doing this.

**Comment by RakanIV** [comment]
I could be wrong on this one but it Looks like not gonna happen anytime soon. I heard in some discord group he's really busy these days. Plus 1.6 stardew is coming but we'll see.

**Comment by Roshan9k** [comment]
> As far as I'd like 1.5 smapi android... 1.6 is literally around the corner.
> 
> It would be like he did all this work for nothing... Literally

yes plus based on the conversation he had before seems like it will be easier for him if he waits for 1.6 game version and 4.0 smapi
